hooks/post/10-post_install.rb

```diff
@@ -15,8 +15,6 @@ def new_install?
   !File.exist?(success_file)
 end
 
-installer_name = kafo.config.app[:installer_name] || kafo.invocation_path
```

Comment:
That change wasn't merged yet when I was working on it, and thought we could just get away with removing this.  But I guess you need to overwrite it to say satellite-installer